app/forms/concerns/wizard.rb

```diff
@@ -0,0 +1,61 @@
+module Wizard
+  extend ActiveSupport::Concern
+
+  included do
+    attr_writer :state_store
+    attr_accessor :current_step, :review_mode
+  end
+
+  def initialize(state_store, params = {})
+    @state_store = state_store
+    sanitize_attributes(params)
+
+    super(last_saved_state.deep_merge(params))
+
+    enter_review_mode! if current_step.eql?('check')
+  end
+
+  def next_step
+    :check
+  end
+
+  def previous_step
+    :new
+  end
+
+  def clear_state!
+    @state_store.delete
+  end
+
+  alias_method :save, :clear_state!
```

Comment:
It was part of the `reasons_for_rejection_wizard.rb` but as I made a typo (it should have been `save!` rather than `save`) and all the tests pass succesfuly I'll just double check to make sure it's not used and remove it.